backend/src/modules/player-profiles/core/application/query/FindAllPlayerProfilesQueryHandler.ts

```diff
@@ -0,0 +1,12 @@
+import {QueryHandler} from "../../../../../shared/core/application/query/QueryHandler";
+import {FindAllPlayerProfiles, FindAllPlayerProfilesResult} from "./FindAllPlayerProfiles";
+import {PlayerProfilesRepository} from "../PlayerProfilesRepository";
+
+export class FindAllPlayerProfilesQueryHandler
```

Comment:
Zapuść sobie `npm run format` i może ustaw sobie IDE, żeby brało pod uwage to formatowanie z prettiera - jak przy poprzednich projektach :) Bo tutaj widze jakieś rozjazdy.

Reply:
done